code/modules/mob/dead/dead.dm

```diff
@@ -71,8 +71,10 @@ INITIALIZE_IMMEDIATE(/mob/dead)
 	if(notransform)
 		return
 	var/list/csa = CONFIG_GET(keyed_list/cross_server)
+	var/list/our_id = CONFIG_GET(string/cross_comms_name)
+	var/list/other = csa.Copy() - our_id

```

Comment:
no need for a copy here, as `-` operators that are not apart of a  `-=` opreator create a copy.

https://secure.byond.com/docs/ref/#/operator/-

> If A is a list, a new list is returned with B removed from the contents of A

Reply:
Oh yeah, not sure what was i thinking, config macro confused me.